src/movement/thruster_sabertooth.cpp

```diff
@@ -84,12 +89,39 @@ void callBack(const robosub::thruster::ConstPtr& msg)
 
 }
 
+void setTimeOut (uint8_t ms_100)
```

Comment:
It's set to a uint8_t because the hardware can't have a value greater than 256, this will allow a compile-time check.